bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -464,6 +467,10 @@ pub struct DecryptedEvent {
     /// is the state of the device at the time of decryption. It may change in
     /// the future if a device gets verified or deleted.
     pub verification_state: VerificationState,
+    /// Safe or Unsafe as per key safety definition.
+    /// Reflects the fact that we are sure that this group key
+    /// is owned by the sending device.
+    pub key_safety: KeySafety,
```

Comment:
Why not make this another variant of the `VerificationState`? I think wrangling two enums is more complex for end users.

Reply:
My idea was to deprecate VerificationState. Compatibility wise for ER, VerificationState is not what we need (mark as untrusted the safe key of a user that we never verified)

Reply:
Also it's a bit orthogonal to Verification. You can have an unsafe key from a trusted sender.
In the future Unsafe keys will be complitely ignored (when assymetric backup and MSC3061 will be deprecated)

Reply:
I changed that to use KeyTrustLevel. 
VerificationState stays for compatibility reason